appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java

```diff
@@ -1217,6 +1217,8 @@ private boolean writeAndroidManifest(File manifestFile) {
         } else if (isMain && isForCompanion) {
           out.write("android:launchMode=\"singleTop\" ");
         }
+        // The line below is required for Android 12+
+        out.write("android:exported=\"true\" ");
```

Comment:
I wonder if it makes sense to only export Screen1. On the other hand, people might have programs that deep link into App Inventor projects in which case every Screen would need to be exported.

Reply:
In order to not break backwards compatibility, let's leave this for now.